assets/stylesheets/comments.scss

```diff
@@ -75,6 +85,34 @@
         padding-left: .25em;
     }
 
+    .openqa-flag {
+        position: relative;
+        background-color: #fdd596;
+        color: #666666;
+        padding: .3em;
+        padding-left: 1.25em;
+        border-radius: .25rem;
+        border: 1px solid #ecd785;
+    }
```

Comment:
4 of these properties are identical to `.openqa-bugref` and `.openqa-label`. Maybe it makes sense to not repeat them and put the common properties into a `.openqa-bugref, .openqa-label, .openqa-flag` block.